app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java

```diff
@@ -31,15 +28,16 @@
 public class CameraViewActivity extends AppCompatActivity implements CameraBridgeViewBase.CvCameraViewListener2 {
     private JavaCameraView preview;
     private PreProcessorFactory ppF;
-    private Boolean diagnoseMode;
-    private long timerDiff;
     private long lastTime;
-    private int numberOfPictures;
-    private int count;
     private String deviceId;
     private String collectionEventId;
     private StudentImageCollectionEventDao studentImageCollectionEventDao;
+    private int count = 1;
```

Comment:
What is the count for? Can we name the variable so that it explains what is being counted?